tests/NuGet.Services.BasicSearchTests/TestSupport/StartedWebApp.cs

```diff
@@ -36,25 +39,29 @@ private StartedWebApp()
 
         private async Task InitializeAsync(IEnumerable<PackageVersion> packages = null)
         {
-            // establish the settings
+            // Establish the settings.
             _settings = ReadFromXml<TestSettings>("TestSettings.xml");
             _nupkgDownloader = new NupkgDownloader(_settings);
             _luceneDirectoryInitializer = new LuceneDirectoryInitializer(_settings, _nupkgDownloader);
             _portReserver = new PortReserver();
 
-            // set up the data
+            // Set up the data.
             var enumeratedPackages = packages?.ToArray() ?? new PackageVersion[0];
             await _nupkgDownloader.DownloadPackagesAsync(enumeratedPackages);
             var luceneDirectory = _luceneDirectoryInitializer.GetInitializedDirectory(enumeratedPackages);
 
-            // set up the configuration
-            var configuration = new InMemoryConfiguration
-            {
-                { "Local.Lucene.Directory", (luceneDirectory as FSDirectory)?.Directory.FullName ?? "RAM" },
-                { "Search.RegistrationBaseAddress", _settings.RegistrationBaseAddress }
-            };
-
-            // set up the data directory
+            // Set up the configuration.
+            // Note that here we are not using DictionaryConfigurationProvider
```

Comment:
`Note that here we are using DictionaryConfigurationProvider`

(remove the "not", it must have been a typo on my part)